.github/workflows/spell-checking.yml

```diff
@@ -0,0 +1,16 @@
+name: Check Spell
```

Comment:
@zhouhanseng please clean up the irrelevant changes! This makes this PR impossible to review

Reply:
sorry for the delay, done.